README.md

```diff
@@ -22,7 +22,23 @@ The code in the authoriser module:
 * Calls the API with the token. If the consignment is returned then the user is authorised and Allow is returned, otherwise Deny is returned.
 
 ### Running Locally
-You can run the Main object in Intellij as you can with any similar project. You will need to provide the program arguments: `export --consignmentId {a uuid}`
+
+Set the `CLIENT_SECRET` environment variable to the client secret for the tdr-backend-checks Keycloak client in the integration environment, which you can find by logging into Keycloak or by running the AWS CLI with intg credentials:
+
+```
+aws ssm get-parameters  --names "/intg/keycloak/backend_checks_client/secret"  --with-decryption
+```
+
+Set the `EXPORT_ROOT_PATH` environment variable to a path on your machine which the task can use to download files and prepare the Bagit package. For example, `/tmp/consignment-export`.
+
+**If you use a Mac**, you will have to do some extra configuration because the export depends on some shell commands which use the Linux default versions:
+
```

Comment:
Really nitpicky here but it's the GNU version as you've put below and not the default for all of Linux. Alpine, doesn't include the GNU versions by default and we had to install them.

Reply:
I didn't realise it might be a wider problem, thanks!

I've moved it to a Troubleshooting section with the actual error that you get, in case it affects anyone else. Do you think the instructions are clear enough that someone could figure out how to fix the problem if they're using a Linux distro without GNU Tar?